adapter/googleServiceControl/googleServiceControl.go

```diff
@@ -0,0 +1,116 @@
+package googleServiceControl
+
+import (
+	"bytes"
+	"context"
+	"github.com/pborman/uuid"
+	sc "google.golang.org/api/servicecontrol/v1"
+	"istio.io/mixer/adapter/googleServiceControl/config"
+	"istio.io/mixer/pkg/adapter"
+	"istio.io/mixer/template/metric"
+	"time"
+)
+
+type handlerBuilder struct {
+	createClientFn
+}
+
+type serviceControlHandler struct {
```

Comment:
Let's call this handler for consistency with other adapters.

Reply:
done